Sources/StreamChatUI/CommonViews/InputTextView/InputTextView.swift

```diff
@@ -98,10 +100,10 @@ open class InputTextView: UITextView, AppearanceProvider {
             )
         )
         placeholderLabel.pin(anchors: [.centerY], to: self)
```

Comment:
Hmm, why do we need this constraint? Aren't we pinning all frame anchors to the superview?

Reply:
I also though it was weird, but we need it some how, otherwise the placeholder is not centered in the y axis 🤔